collector/app/core/models/transaction.py

```diff
@@ -0,0 +1,36 @@
+"""Module that includes functionality to work with transaction data."""
+
+import logging
+
+from asyncpg import exceptions
+
+from core.database.queries import INSERT_TRANSACTION
+
+
+LOG = logging.getLogger(__name__)
+
+
+class Transaction:
+    """Model that provides methods to work with transaction data."""
+
+    def __init__(self, postgres=None, redis=None):
+        """Initialize transaction instance with required clients."""
+        self.postgres = postgres
+        self.redis = redis
```

Comment:
Maybe these attributes should be private?

Reply:
Make sense.